utils/node_local_test.py

```diff
@@ -437,23 +453,15 @@ def stop(self):
         assert rc.returncode == 0
 
         start = time.time()
+        max_stop_time = 1
```

Comment:
1 or 20?  This seems a big decrease but I'm not online to check how long it typically takes.  As it's a timeout, not a sleep I'd be inclined to error on the side of caution.

Reply:
I had dropped it to 1 for some testing, forgot to bring it back up. I do think 20s is excessive, just changed it to 5.